docker/relayerConfigTemplate.json

```diff
@@ -2,6 +2,7 @@
     "network-id": 1337,
     "p-chain-api-url": "http://127.0.0.1:9650",
     "encrypt-connection": false,
+    "process-missed-blocks": false,
```

Comment:
Do we need to add the "startBlock" to the source subnet configurations here?

Reply:
If `process-missed-blocks` was true, then yes we would have to do that. But the starting block is only calculated is `process-missed-blocks` is enabled